Vagrantfile

```diff
@@ -4,28 +4,28 @@
 Vagrant.configure("2") do |config|
 
   #
-  # fake a managed server by bringing up a virtualbox vm
+  # fake a managed linux server by bringing up a virtualbox vm
   #
-  config.vm.define :fake_managed_server do |fms_config|
+  config.vm.define :fake_managed_linux_server do |fms_config|
```

Comment:
FWIW, I named these local_linux and managed_linux in my example and it has made it easier for me to tell them apart.